ui/page/settings_page.go

```diff
@@ -440,62 +366,26 @@ func (pg *SettingsPage) HandleUserInteractions() {
 		break
 	}
 
+	for pg.backButton.Button.Clicked() {
+		pg.ParentNavigator().CloseCurrentPage()
+	}
+
 	for pg.currency.Clicked() {
 		currencySelectorModal := preference.NewListPreference(pg.Load,
 			dcrlibwallet.CurrencyConversionConfigKey, values.DefaultExchangeValue,
 			values.ArrExchangeCurrencies).
-			Title(values.StrCurrencyConversion).
+			Title(values.StrExchangeRate).
 			UpdateValues(func() {})
 		pg.ParentWindow().ShowModal(currencySelectorModal)
 		break
 	}
 
-	if pg.isDarkModeOn.Changed() {
-		pg.WL.MultiWallet.SaveUserConfigValue(load.DarkModeConfigKey, pg.isDarkModeOn.IsChecked())
+	for pg.appearanceMode.Clicked() {
+		pg.isDarkModeOn = !pg.isDarkModeOn
+		pg.WL.MultiWallet.SaveUserConfigValue(load.DarkModeConfigKey, pg.isDarkModeOn)
 		pg.RefreshTheme(pg.ParentWindow())
 	}
 
-	if pg.spendUnconfirmed.Changed() {
-		pg.WL.MultiWallet.SaveUserConfigValue(dcrlibwallet.SpendUnconfirmedConfigKey, pg.spendUnconfirmed.IsChecked())
-	}
-
-	if pg.governance.Changed() {
-		if pg.governance.IsChecked() {
-			go pg.WL.MultiWallet.Politeia.Sync()
-			pg.WL.MultiWallet.SaveUserConfigValue(load.FetchProposalConfigKey, pg.governance.IsChecked())
-			pg.Toast.Notify(values.StringF(values.StrPropFetching, values.String(values.StrEnabled), values.String(values.StrCheckGovernace)))
-		} else {
-			info := modal.NewInfoModal(pg.Load).
-				Title(values.String(values.StrGovernance)).
-				Body(values.String(values.StrGovernanceSettingsInfo)).
-				NegativeButton(values.String(values.StrCancel), func() {}).
-				PositiveButtonStyle(pg.Theme.Color.Surface, pg.Theme.Color.Danger).
-				PositiveButton(values.String(values.StrDisable), func(isChecked bool) bool {
-					if pg.WL.MultiWallet.Politeia.IsSyncing() {
-						go pg.WL.MultiWallet.Politeia.StopSync()
-					}
-					pg.WL.MultiWallet.SaveUserConfigValue(load.FetchProposalConfigKey, !pg.governance.IsChecked())
-					pg.WL.MultiWallet.Politeia.ClearSavedProposals()
-					pg.Toast.Notify(values.StringF(values.StrPropFetching, values.String(values.StrDisabled)))
-					return true
-				})
-			pg.ParentWindow().ShowModal(info)
-		}
-	}
-
-	if pg.beepNewBlocks.Changed() {
-		pg.WL.MultiWallet.SaveUserConfigValue(dcrlibwallet.BeepNewBlocksConfigKey, pg.beepNewBlocks.IsChecked())
-	}
-
-	if pg.proposalNotification.Changed() {
-		pg.WL.MultiWallet.SaveUserConfigValue(load.ProposalNotificationConfigKey, pg.proposalNotification.IsChecked())
-		if pg.proposalNotification.IsChecked() {
-			pg.Toast.Notify(values.StringF(values.StrPropNotification, values.String(values.StrEnabled)))
-		} else {
-			pg.Toast.Notify(values.StringF(values.StrPropNotification, values.String(values.StrDisabled)))
-		}
-	}
-
 	if pg.transactionNotification.Changed() {
 		pg.WL.MultiWallet.SaveUserConfigValue(load.TransactionNotificationConfigKey, pg.transactionNotification.IsChecked())
```

Comment:
There is a slight lag between when `Transaction notification` switch is clicked and when it is enabled/disabled. This is because writing to the config db is fairly expensive operation. 

https://github.com/planetdecred/godcr/blob/3fafcddf2577a4009e135ff5d4d339d9def068fd/ui/page/settings_page.go#L390

Run this code in a goroutine to prevent lag.
`go func() {
    		pg.WL.MultiWallet.SaveUserConfigValue(load.TransactionNotificationConfigKey,     pg.transactionNotification.IsChecked())
}()`